src/test/java/bio/terra/service/snapshot/SnapshotTest.java

```diff
@@ -285,4 +298,66 @@ public void snapshotByFullViewHappyPathTest() throws Exception {
         assertEquals("new snapshot has been created", snapshot.getName(), requestModel.getName());
         assertEquals("all 5 relationships come through", snapshot.getRelationships().size(), 5);
     }
+
+    @Ignore("We expect this test to be flaky b/c ACLs can take a while to update (up to 7 minutes)")
+    @Test
+    public void snapshotACLTest() throws Exception {
+        DatasetModel dataset = dataRepoFixtures.getDataset(steward(), datasetId);
+
+        String projectId = System.getenv("GOOGLE_CLOUD_DATA_PROJECT");
+        bigQuery = BigQueryOptions.newBuilder()
+            .setProjectId(projectId)
+            .build()
+            .getService();
+
+        // Fetch BQ Dataset
+        String datasetName = dataset.getName();
+        String bqDatasetName = bigQueryPdao.prefixName(datasetName);
+        Dataset bq_dataset = bigQuery.getDataset(bqDatasetName);
+
+        // fetch ACLs
+        List<Acl> beforeAcls = bq_dataset.getAcl();
+        assertEquals("There should be 7 ACLs on the dataset", 7, beforeAcls.size());
+        logger.info("---- Dataset ACLs before snapshot create-----");
+        beforeAcls.forEach(acl -> {
+            logger.info("Acl: {}", acl.toString());
```

Comment:
`toString()` unnecessary here
```suggestion
            logger.info("Acl: {}", acl);
```